tensorflow/python/keras/engine/training.py

```diff
@@ -1434,6 +1441,7 @@ def evaluate_generator(self,
         workers=workers,
         use_multiprocessing=use_multiprocessing,
         verbose=verbose,
+        class_weight=class_weight,
```

Comment:
evaluate_generator() is missing the class_weight param, which cause test failure. Please check the test log.

Reply:
I added the class_weight param in evaluate_generator()